### DISCUSSION TITLE
Welcome to Exomiser Discussions!

### DISCUSSION THREAD

**Discussion by julesjacobsen** [exomiser/Exomiser#423] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * 📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.
  * We will trial using Discussions as a means of announcing new data releases. These are vital to enable the best performance of the software, so keep an eye out.
   
  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by squinker** [comment]
Hi, are you accepting Merge Requests at the moment?

I'm interested in building Exomiser locally but the build fails because of these two dependencies:

```
<dependency>
            <groupId>de.sonumina</groupId>
            <artifactId>javautil</artifactId>
            <version>0.0.1</version>
        </dependency>
        <dependency>
            <groupId>ontologizer</groupId>
            <artifactId>ontologizer</artifactId>
            <version>0.0.1-SNAPSHOT</version>
        </dependency>
```

Are these available somewhere else?

Thanks

**Comment by pnrobinson** [comment]
@julesjacobsen we should try to replace these dependencies, they are no longer supported and probably phenol can do whatever we need.

**Reply by squinker** [reply]
yep that's certainly what the comment in the code says. I'm just curious how people build this locally, do people have cached copies somewhere?

**Comment by julesjacobsen** [comment]
The repo hosting these has recently been decommissioned and it's not our repo. I hadn't noticed as I have the artifacts in my local m2 repository. You can untar these files: [sonumina.tar.gz](https://github.com/exomiser/Exomiser/files/9811579/sonumina.tar.gz)
 into your local .m2 directory and the build should work.  

I'll be removing the dependencies in the next release.

**Reply by squinker** [reply]
Ah thanks Jules.

**Reply by squinker** [reply]
Thanks for the tar, I did cp -r .m2/repository/ ~/.m2/repository and I can compile now.
I got 


[ERROR] java.lang.ClassNotFoundException: org.apache.maven.surefire.junitplatform.JUnitPlatformProvider
[ERROR] org.apache.maven.surefire.booter.SurefireBooterForkException: There was an error in the forked process
[ERROR] java.lang.ClassNotFoundException: org.apache.maven.surefire.junitplatform.JUnitPlatformProvider

But changing the version of maven-surefire-plugin from 3.0.0-M5 to 2.22.2 fixed this

**Comment by patrickrshea** [comment]
Is there a way to provide a list of variant IDs for Exomiser to blacklist during a run?  I've looked through the documentation and source code and saw the ability to whitelist variants, but can I do the opposite?  I'm trying to exclude a list of known artifact variants that keep showing up in the output.

**Comment by damiansm** [comment]
Hi Patrick,

There is no specific variant blacklist filter but you can do local allele
frequency filtering which will have the same effect. If you don't have the
actual local frequencies in your cohort for these variants you could just
produce a file with some dummy MAF like 50% and configure
application.properties for this file and set LOCAL in the list of filtering
sources

Best wishes
Damian
